api/query/cache/service/main.go

```diff
@@ -158,12 +159,25 @@ func main() {
 	var netClient = &http.Client{
 		Timeout: time.Second * 5,
 	}
+
+	// Initializes Logger
+	var gcClient *gclog.Client
+
+	gcClient, err = gclog.NewClient(context.Background(), *projectID)
+	if err != nil {
+		logrus.Fatalf("Failed to initiate gclog: %v", err)
+	}
+	defer gcClient.Close()
+
+	childrenLogger := gcClient.Logger("request_logs", gclog.CommonResource(&monitoredResource))
+	parentLogger := gcClient.Logger("request", gclog.CommonResource(&monitoredResource))
```

Comment:
```suggestion
	parentLogger := gcClient.Logger("request_log", gclog.CommonResource(&monitoredResource))
```
(trying to follow the naming convention of App Engine standard here)